src/codegate/api/v1.py

```diff
@@ -665,3 +671,89 @@ async def get_workspace_token_usage(workspace_name: str) -> v1_models.TokenUsage
     except Exception:
         logger.exception("Error while getting messages")
         raise HTTPException(status_code=500, detail="Internal server error")
+
+
+@v1.get("/personas", tags=["Personas"], generate_unique_id_function=uniq_name)
+async def list_personas() -> List[Persona]:
+    """List all personas."""
+    try:
+        personas = await dbreader.get_all_personas()
```

Comment:
why not leverage the persona manager here to keep the code consistent? Adding the dbreader here seems like an antipattern